src/Nethermind/Nethermind.Trie.Benchmark/Nethermind.Trie.Benchmark.csproj

```diff
@@ -5,7 +5,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="BenchmarkDotNet" Version="0.13.2" />
+      <PackageReference Include="BenchmarkDotNet" />
```

Comment:
Sneaked in this change since it was causing constant issues with my setup: the IDE would complain about failures when restoring projects, and the CLI would refuse to build or run tests (`dotnet build`, `dotnet test`, etc).